bin/run-utils.js

```diff
@@ -184,3 +184,195 @@ function runFirefox(options) {
   });
 }
 exports.runFirefox = runFirefox;
+
+var currentSample = null;
+var currentSampleMap = null;
+var prefsRE = /^user_pref\(['"](.*)["'],\s*['"]?(.*)["']?\)/;
+
+function parsePrefValue(value) {
+  if (/^[\d]+$/.test(value)) {
+    return parseInt(value, 10);
+  }
+  if (value === "true") {
+    return true;
+  }
+  if (value === "false") {
+    return false;
+  }
+  return value;
+}
+
+function takePrefsSample(profilePath) {
```

Comment:
What do you mean by Sample? This looks to read in the whole prefs file as opposed to some of it. There's various things that make this function somewhat hard to read, and we might not actually need all the complexity anyway. See comment for setUserPrefs.